awscurl/awscurl.py

```diff
@@ -433,6 +436,8 @@ def inner_main(argv):
     # https://github.com/boto/botocore/blob/c76553d3158b083d818f88c898d8f6d7918478fd/botocore/credentials.py#L260-262
     parser.add_argument('--security_token', env_var='AWS_SECURITY_TOKEN')
     parser.add_argument('--session_token', env_var='AWS_SESSION_TOKEN')
+    parser.add_argument('-f', '--follow-redirects', action='store_true', default=False, 
```

Comment:
in spirit of following curl, please see `curl --help | grep Follow` -- i think awscurl should follow curl's convention